tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -1181,6 +1181,8 @@ class CudnnRnnDescriptor : public dnn::RnnDescriptor {
     }
 
     // Create the params handle.
+    // TODO(kaixih): Can be removed when cudnnRNNForward*** and
+    // cudnnRNNForward***Ex are deprecated.
```

Comment:
I could be missing something, but why is removing something dependent on deprecating some API?  Do you mean to to say "Can be removed when alternatives for API XYZ is available."?

Reply:
This is meant to say that "We can remove this when cudnnRNNForward*** and cudnnRNNForward***Ex are removed from the code since the new API doesn't need it."